#This message allows to receive the state of the hardware of the robot
int16 imu_value
uint8 free_wheel_activated
uint8 ball_sensor
int16 encoder_1
int16 encoder_2
int16 encoder_3
float32 battery_pc
float32 battery_camera
float32 battery_main
